client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-add-license-notification/style.scss

```diff
@@ -10,24 +10,35 @@
 }
 
 .site-add-license-notification__license-banner-wp {
-	.banner {
+	.banner.card {
```

Comment:
Added for better targeting since some styles were not getting applied sometimes.